packages/deployments/contracts/contracts/core/connext/helpers/RelayerProxy.sol

```diff
@@ -288,6 +271,19 @@ contract RelayerProxy is ProposedOwnable, ReentrancyGuard, GelatoRelayFeeCollect
 
   // ============ External Functions ============
 
+  /**
+   * @notice Indicates if the job is workable by the sender. Takes into account the Autonolas priority.
+   * For example, if priority is 3, then Autonolas will be able to work on blocks 0, 1, 2, and 3.
+   * @param sender The address of the caller
+   */
+  function isWorkableBySender(address sender) public view returns (bool) {
+    if (sender == autonolas) {
+      return block.number % 10 <= autonolasPriority;
+    } else {
+      return block.number % 10 > autonolasPriority;
+    }
+  }
```

Comment:
What do you think about making it a modifier, which contains the error as well?
One more thing, the `sender` arg should be named `_sender`